sdk/servicebus/service-bus/test/internal/shared.spec.ts

```diff
@@ -30,4 +31,45 @@ describe("shared", () => {
       `An error was thrown from the user's processError handler: Error: Whoops!`
     ]);
   });
+
+  it("getMessageIterator doesn't yield empty responses", async () => {
+    const messages = [
+      [],
+      [
+        {
+          id: "hello"
+        }
+      ]
+    ];
+
+    const receiver: Pick<ServiceBusReceiver<any>, "receiveMessages"> = {
+      receiveMessages: async (maxMessageCount, _options) => {
+        assert.equal(maxMessageCount, maxMessageCount);
```

Comment:
What is this assertion for?

Reply:
So that TS won't complain about not using maxMessageCount?

Reply:
Oh no. Total typo.